### PR TITLE
Bug 1131243 - Add private browsing notification in rocketbar

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1131243

### CODE REVIEW

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=630278557120)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0167c9096c10)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a678000d9f2d)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7d91aea8e684)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=111c17a95c3f)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=93caed581221)

File: apps/system/style/chrome/chrome.css [hunk 1]
```diff
@@ -232,6 +232,24 @@
...
+  -moz-padding-start: 4rem;
+}
+
+.appWindow.private > .chrome.maximized .pb-icon {
+  display: block;
+  width: 4rem;
+  height: 100%;
+  position: absolute;
+  top: 0.1rem;
+  left: 0.3rem;
```
Comment (mikehenrty) [inline apps/system/style/chrome/chrome.css]: Need to specify `right: 0.3rem` to make this work in RTL mode.

Comment (KevinGrandon) [inline apps/system/style/chrome/chrome.css]: Ah nice catch, suppose I shouldn't be breaking RTL at this point :)

Comment (KevinGrandon) [inline apps/system/style/chrome/chrome.css]: Added as next line.

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=3237bc3c01fb)

**Comment by try-server-hook** [comment]
[KevinGrandon Kevin Grandon (KevinGrandon)](https://github.com/KevinGrandon) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ccaf73d07a04)

**Pull request merged**